[TimeSeriesInsights] Onboard with track2 SDK (#2817)
Azure/azure-cli-extensions: src/timeseriesinsights/HISTORY.rst

Release History
===============

0.2.0
++++++

> * Breaking Change: use track2 SDK

Reviewer: This doesn't seem like user-facing change. Maybe we can put actual changes here?

Release History
===============

0.2.0
++++++

> * [BREAKING CHANGE] Deprecate ``az timeseriesinsights`` command group, add a command group ``az tsi``

Reviewer: ```suggestion
* [BREAKING CHANGE] ``az timeseriesinsights`` command group is replaced by ``az tsi``
```

* ``az tsi environment`` support ``gen1 environment`` and ``gen2 environment``

> * Add command group `az tsi reference-data-set`

Reviewer: ```suggestion
* Add command group ``az tsi reference-data-set``
```
